### PR TITLE
Make desktop responsive

### PR DESCRIPTION
**Pull request by fdezcaminero**
* Mobile version and desktop version are up and running
* The design changes significantly and harmoniously from one screen to the other
* I had a problem with the branch I was working on. It was 22 commits behind from main. To fix this I created this branch from main and copy pasted the code from the branch I was working on. For my commit history refer to this link: https://github.com/fdezcaminero/figmaPortfolio2023/commits/create-desktop-version (I think this happened because at one point I had an incident where I thought I had lost all my work, and had to copy paste what I had saved, and created a new branch from a main that was not up to date with the remote repo).
* Desktop version looks best on larger screens (1440px>)
* Buttons have hover and pressed effects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,8 +91,8 @@ <h2 class="playfairFont projectsTag">Projects</h2>
         </section>
 
         <section class="flex-column">
-          <img class="fullWidth" src="resources/ABlaptop.png" alt="A laptop and a phone">
-          <div class="darkBlueBackground projectsPadding flex-column">
+          <img class="fullWidth ABLaptopPic" src="resources/ABlaptop.png" alt="A laptop and a phone">
```
Comment (AbelG101) [inline index.html:94]: - [ ] You were so close, but as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=STzDZgG5tHGW7XUO-0) the width of this image should take around 50% of the screen's width, and the mac book image above should stick above it. Kindly make sure you fix it ✔. 
     - Figma design:
        image
     - Your design:
        Inkedfirefox_ENU6HMj2nd

File: index.html [hunk 2]
```diff
@@ -200,13 +224,21 @@ <h2 class="playfairFont">Get Started</h2>
     <p class="loremStyle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
```
Comment (AbelG101) [inline index.html:224]: - [ ] Please make sure the elements in the contact-me section are aligned properly as specified in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=STzDZgG5tHGW7XUO-0). 
- [ ] **Hint:** You can add a separate container for the heading, the text, and the form and give it a limited width and align it to the center. The heading should also be aligned to the center.
    - Figma design:
      gRLsROsxoT
    - Your design:
      image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @fdezcaminero 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- UI is looking on point ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- I'm really loving your portfolio here you're doing an amazing job 💯, you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @fdezcaminero  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Made suggested changes 👍 
✔ No linter Errors👍 
✔ awesome design👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**